mmv1/products/notebooks/terraform.yaml

```diff
@@ -116,6 +116,53 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
       shieldedInstanceConfig: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  Runtime: !ruby/object:Overrides::Terraform::ResourceOverride
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 15
+      update_minutes: 15
+      delete_minutes: 15
+    autogen_async: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_basic"
+        primary_resource_id: "runtime"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime%s\", context[\"random_suffix\"])"
```

Comment:
Is there a reason you need this fmt.Sprintf here?

if you include a dash in the resource name it should auto append a random suffix

Reply:
Mostly copy/paste tbh. Where does the dash go? I couldn't find a clear example of that.

Reply:
you can just call the resource "notebooks-runtime"

Reply:
iam_notebooks_runtime_generated_test.go doesnt seem to like what I am doing. Do you have any example by any chance?

Reply:
Ah, I was missing the \" in my tries. Duh. 
Done.

Reply:
Actually, no. The IAM generated tests complain that they don't see the suffix for some reason and only look for `tf-test-notebooks-runtime-` or `tf-test-notebooks-runtime` if I don't use the hyphen.